synapse/rest/media/v1/preview_url_resource.py

```diff
@@ -187,7 +187,8 @@ def _do_preview(self, url, user, ts):
         if _is_media(media_info['media_type']):
             file_id = media_info['filesystem_id']
             dims = yield self.media_repo._generate_thumbnails(
-                None, file_id, file_id, media_info, url_cache=True,
+                None, file_id, file_id, media_info["media_type"],
```

Comment:
would be nice to factor out a local `media_type` here too, given the number of times it's referenced, but shrug